mathesar/serializers.py

```diff
@@ -9,7 +9,7 @@ class NestedTableSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Table
-        fields = ['id', 'name', 'url']
+        fields = ['id', 'oid', 'name', 'url']
```

Comment:
I'm not sure we should expose the `oid` in the API, it seems unnecessary. We can always add it later if it's useful.

Reply:
Makes sense.